src/js/components/recipient/RecipientPage.jsx

```diff
@@ -32,29 +33,45 @@ export default class RecipientPage extends React.Component {
         super(props);
 
         this.state = {
-            showModal: false
+            showChildRecipientModal: false,
+            showAlternateNamesRecipientModal: false
         };
 
-        this.showModal = this.showModal.bind(this);
-        this.hideModal = this.hideModal.bind(this);
+        this.showChildRecipientModal = this.showChildRecipientModal.bind(this);
+        this.hideChildRecipientModal = this.hideChildRecipientModal.bind(this);
+        this.showAlternateNamesRecipientModal = this.showAlternateNamesRecipientModal.bind(this);
+        this.hideAlternateNamesRecipientModal = this.hideAlternateNamesRecipientModal.bind(this);
     }
 
-    showModal() {
+    showAlternateNamesRecipientModal() {
```

Comment:
For future reference, ES6 arrow functions automatically bind this. Therefore there is no need to bind in the constructor if you did this `showAlternateNamesRecipientModal = () => {}`